bin/speed-tweet

```diff
@@ -22,4 +22,4 @@ if(argv.help){
 }
 
 var recurTest = require('../lib/recur-test')
-recurTest(argv.t, argv.r)
+recurTest(argv.r, argv.t, argv.i)
```

Comment:
Thanks for catching my typo! Looks like the order of arguments should still be swapped though:

```javascript
function recur(time, total, recipient){
/* code */
}
```
^ [Original File](https://github.com/tribet84/speed-and-tweet/blob/master/lib/recur-test.js#L5)

`argv.r` - Should have recipient
`argv.i` - Should have interval, which is specified as `time` in the parameters.